api/pandas/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Pandas
+nav_order: 3
+parent: API
+has_children: true
+---
+# Data Commons Pandas API
+
+The **Data Commons Pandas API** is a superset of the Data Commons Python API:
+all functions from the Python API are also accessible from
+the Pandas API, and supplemental functions help with directly creating pandas
```

Comment:
nit: add link for "pandas", and should it be Pandas?

Reply:
pandas is technically lower case, but I've upper cased it for the "Pandas API".  I added a link.